artifacts/scripts/util.sh

```diff
@@ -757,6 +757,11 @@ function update_full_godeps() {
     echo "Running godep save."
     godep save ./...
 
+    # restore all other files of Godeps/ (like OWNERS), but preserve the new Godeps.json
+    cp Godeps/Godeps.json Godeps/Godeps.json.preserve
```

Comment:
So this basically moves the Godeps.json to an untracked file, then re-checks out the dir from HEAD, then moves the untracked file back overtop the correct filename? I haven't dug too much into the publisher code.

Reply:
yes. Godeps does not like the have an existing Godeps/ dir. So we remove it. But then we lost Godeps/OWNERS. Any following commit changing OWNERS will break (if we don't preserve it).

Reply:
Right. Okay, this should work then. A very duct tape solution, but 🤞 we won't have to deal with Godeps too much longer.

Reply:
The scripts is 50% duct-tape around godep.